TODO.md

```diff
@@ -0,0 +1,6 @@
+# TODO list
+
+- add version catalog / platform project
+- add code formatter
+- create a screen in Flutter
```

Comment:
let's remove this line, we didn't decide this yet

Reply:
5d81519b50c2eb5d9f315d4f682aa4c3b81cd434